docs/conf.py

```diff
@@ -92,7 +92,7 @@
 # General information about the project.
 project = 'MetPy'
 # noinspection PyShadowingBuiltins
-copyright = '2019, MetPy Developers'
+copyright = '2019, MetPy Developers. Development supported by the National Science Foundation.'
```

Comment:
What do we think about adding either/both the NSF logo and the award numbers?

Reply:
Easy to add the award number here (I don't have it, so if you could reply with that). Not sure how adding the logo in the footer goes. Would be easier to add it to the index page I'm guessing.